WordPressKit/WordPressKit/PluginDirectoryEntry.swift

```diff
@@ -17,6 +17,15 @@ public struct PluginDirectoryEntry: Equatable {
     private let faqHTML: String?
     private let changelogHTML: String?
 
+    public var descriptionText: NSAttributedString? { return extractHTMLText(self.descriptionHTML) }
+    public var installationText: NSAttributedString? { return extractHTMLText(self.installationHTML) }
+    public var faqText: NSAttributedString?  { return extractHTMLText(self.faqHTML) }
+    public var changelogText: NSAttributedString? { return extractHTMLText(self.changelogHTML) }
+
+    private let rating: Int
+    public var starRating: Double { return (Double(rating) / 10).rounded() / 2 } // rounded to nearest half.
```

Comment:
Probably a good idea to move the closure into a new line, for readability purposes

Reply:
(Same in the calculated properties above)